### PR TITLE
css docs

### PR DESCRIPTION
**Pull request by xi**
This fixes some css documentation. Now nearly everything should be documented.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_user_indicator.scss [hunk 1]
```diff
@@ -7,6 +7,23 @@ category: Widgets
...
+    </div>
+</div>
+<div class="user-indicator">
+    <img src="/static/images/sample.jpg" alt="" class="user-profile-image" />
+    <adh-user-profile-image data-path="{{credentials.userPath}}"></adh-user-profile-image>
+    <a href="#" class="user-indicator-name">John Doe</a
+    ><span class="user-indicator-name-logout-separator">,</span>
+    <a class="user-indicator-logout" href="#">logout</a>
+</div>
+```
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_user_indicator.scss]: nice.

**Comment by der-john** [comment]
I stumbled across the 'disguised link mixin'. As far as I can see, it's not used anywhere. Should I remove it?

**Comment by xi** [comment]
> I stumbled across the 'disguised link mixin'. As far as I can see, it's not used anywhere. Should I remove it?

It is actually used in comment.scss

**Comment by der-john** [comment]
> I stumbled across the 'disguised link mixin'. As far as I can see, it's not used anywhere. Should I remove it?
It is actually used in comment.scss

oh. thank you.

**Comment by der-john** [comment]
looks good!

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss [hunk 2]
```diff
@@ -44,8 +44,8 @@ depends on a lot of factors:
     `password_reset`) do not have a header.
 -   Areas related to user management (`login`, `register`,
     `create_password_reset`) have a simplified header.
--   The header in the resource area may have additional, process related
-    functionality.
+-   The header above a process or proposal page may have additional,
+    process-related functionality.
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss]: I liked the previous version better.

-  we do not have a clear concept of a "page"
-  the header on a comment resource may also contain process-related functionality.

**Pull request merged**